packages/s/soroban-client.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "soroban-client",
+  "description": "soroban-client is a library for working with the Soroban RPC server.",
+  "filename": "soroban-client.min.js",
+  "keywords": [
+    "soroban",
+    "stellar",
+    "helper",
+    "hash",
+    "xdr"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stellar/js-soroban-client.git"
+  },
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "npm",
+    "target": "soroban-client",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
👋 We try to avoid using globstars, especially open ones, as they can lead to matching files that we didn't intend to match. Please can you specify the directories and extensions that you want to match here -- looking at the CI run, probably just `*.js` would do.